### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by utsargo**
Added a responsive mobile menu as required.

- The mobile menu is only visible in a viewport less than 768px wide
- Used javascript methods like querySelector, classList, toggle, addEventListener, etc.
- Used CSS to build and transform the hamburger icon.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
> Hi @utsargo,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
## Invalid Code Review Request🛑 

>- [x] Kindly update your linters.yml file for this project to include ESLinters. In the root project, you have a folder named .github. Inside this one, you have another folder named workflows. And inside the workflows folder, you have a file named linters.yml. Inside this file, you need to update the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js);

image

> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

File: assets/styles/style.css [hunk 1]
```diff
@@ -130,6 +138,42 @@ section {
...
+  text-decoration: none;
+  color: #344563;
+  font-size: 18px;
+  font-weight: 600;
+  transition: 0.3s ease-out;
+}
+
+.navbar .main-nav .main-menu .nav-item .nav-link:hover {
+  color: #36b37f;
+}
```
Comment (levy002) [inline assets/styles/style.css:179]: - [ ] Kindly make sure your designs look exactly like the provided Figma design.

  - ### YOUR DESIGN
     image

  - ### FIGMA DESIGN
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @utsargo   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @utsargo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**